Sources/ApolloCodegenLib/ApolloCodegen.swift

```diff
@@ -67,6 +67,22 @@ public class ApolloCodegen {
       }
     }
   }
+  
+  public struct CodeGenerationBuildOptions: OptionSet {
```

Comment:
These are more "Types of things to generate" rather than "build options". I do like the approach, but I think we should change the name maybe.

I could see us in the future altering this to allow you to say only re-generate fragments, or even only regenerate models in a specific sub-folder or something also.

Reply:
Yea I was having a hard time coming up with a good name for this, but figured the overall approach could have some benefits so I am open to different names for this.